components/script/dom/webglrenderingcontext.rs

```diff
@@ -1399,6 +1399,25 @@ impl WebGLRenderingContextMethods for WebGLRenderingContext {
         buffer.map_or(false, |buf| buf.target().is_some() && !buf.is_deleted())
     }
 
+    // https://www.khronos.org/registry/webgl/specs/latest/1.0/#5.14.3
+    fn IsEnabled(&self, cap: u32) -> bool {
+        match cap {
+            constants::BLEND | constants::CULL_FACE | constants::DEPTH_TEST | constants::DITHER |
+            constants::POLYGON_OFFSET_FILL | constants::SAMPLE_ALPHA_TO_COVERAGE | constants::SAMPLE_COVERAGE |
+            constants::SAMPLE_COVERAGE_INVERT | constants::SCISSOR_TEST => {
+                let (sender, receiver) = ipc::channel().unwrap();
```

Comment:
Can you write a comment here in the lines of:

```
// TODO: We could write this without the need of IPC if needed, recording the calls to `enable` and `disable`.
```

This is more than fine for now though, thanks for doing this!

r=me with that.